src/main/groovy/com/athaydes/gradle/ceylon/util/CeylonCommandOptions.groovy

```diff
@@ -57,6 +61,15 @@ class CeylonCommandOptions {
         return getCommonOptions( project, config ) + options
     }
 
+    static List getFatJarOptions(Project project,CeylonConfig config){
+        def options = [ ]
+        def out = getFatOut(project, config)
+        if(out!=null){
+            options << "--out=$out.absolutePath}"
+        }
+        return options
+    }
```

Comment:
Please let your IDE auto-format the code so the format is consistent everywhere. Notice the lack of whitespaces in many places.